analytic_schedule/tests/test_account_analytic_account.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+from odoo import fields
+
+
+class TestAccountAnalyticAccount(TransactionCase):
+
+    def setUp(self):
+        super(TestAccountAnalyticAccount, self).setUp()
+        self.analytic_account_obj = self.env['account.analytic.account']
+        self.analytic_line_obj = self.env['account.analytic.line']
+        self.partner1 = self.env.ref('base.res_partner_1')
+        self.partner2 = self.env.ref('base.res_partner_2')
+        self.analytic_parent = self.analytic_account_obj.create({
+            'name': 'Test Parent',
+            'code': '01',
+        })
```

Comment:
The date should be here, not in the son. Then, when creating the son don't assign any date as the system is supposed to do so.

Regards